### DISCUSSION TITLE
Release and Support Schedules

### DISCUSSION THREAD

**Discussion by buckaroogeek** [moby/moby#48587] [Q&A] [answered]
I have been looking for moby related release and support schedules. More specifically is there a schedule for new releases (v28, v29 and so on)? And is there a schedule showing when each release (v23, v25, etc) reaches end of support?  Thanks!

**Comment by AkihiroSuda** [comment, marked as answer]
> And is there a schedule showing when each release (v23, v25, etc) reaches end of support?

Here: https://github.com/moby/moby/blob/master/project/BRANCHES-AND-TAGS.md

> 
> | Branch Name                       | Sponsoring Maintainer(s)                                | Contribution Status   | Expected End of Maintenance  | Known Distributors                  |
> |-----------------------------------|---------------------------------------------------------|-----------------------|------------------------------|-------------------------------------|
> | master (development branch)       | The Moby Project [MAINTAINERS](../MAINTAINERS)          | N/A                   | -                            | N/A                                 |
> | 27.x                              | The Moby Project [MAINTAINERS](../MAINTAINERS)          | Maintained            | After 28.x                   | [Docker, Inc.](https://docker.com)  |
> | 26.1                              | [@]cpuguy83                                               | Maintained (security) | TBD                          | [Microsoft](https://microsoft.com)  |
> | 25.0                              | [@]corhere                                                | Maintained            | TBD                          | [Amazon][al2023]                    |
> | 23.0                              | [@]corhere                                                | Maintained            | [2025-05-19][mcr23-maint]    | [Mirantis](https://mirantis.com)    |
> 
> [al2023]: https://docs.aws.amazon.com/linux/
> [mcr23-maint]: https://docs.mirantis.com/mcr/23.0/compat-matrix/maintenance-lifecycle.html
>

**Comment by buckaroogeek** [comment]
Apologies for the delayed reply. This is very helpful. Am I correct in concluding that the v27.x Branch is now in Unmaintained Contributor Status?

thank you

**Reply by thaJeztah** [reply]
Correct; 

- https://github.com/moby/moby/pull/49910
